versioned_docs/version-2.7/integrations-in-rancher/cloud-marketplace/azure-marketplace-payg-integration/installing-rancher-prime.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Installing Rancher Prime PAYG on Azure
+---
+
+This page covers installing the Rancher Prime PAYG offering on Microsoft's Azure Marketplace.
+
+## How to Install Rancher Prime PAYG
+
+Refer to the following steps for creating a new deployment of Rancher Prime from the Azure Marketplace page.
+
+1. Select the **Rancher Prime with 24x7 Support** offer (either **EU and UK only** or **non-EU and non-UK only**) that corresponds to the location where your account is registered.
+1. Choose the plan from the dropdown list. View the **Plans + Pricing** tab for more details about the plan.
+1. Select **Create**.
+
+### Basics
+
+On the **Basics** tab, specify the **Project details** and **Instance details**:
+
+1. Select an existing **Subscription** from the dropdown list.
+1. Select an existing **Resource group** from the dropdown list.
+
+   :::note
+
+   The **Create new** resource group feature is not supported.
+
+   :::
+   ![Create new resource group not supported](/img/install-rancher-prime-basics-create-new.png)
+1. Select an existing **AKS Cluster Name** from the dropdown list.
+1. Choose an **Extension Resource name**. It can consist of alphanumeric characters and dots; the length must be between 2 and 253 characters.
+
+   ![Basics tab](/img/install-rancher-prime-basics.png)
+1. Select **Next**.
+
+### Rancher Configuration
+
+On the **Rancher Configuraion** tab, specify the following information:
+
+1. Enter the **Hostname** for Rancher. The Rancher hostname must be a fully qualified domain name (FQDN) and the Rancher server URL will be created using this hostname.
+   :::note
+
+   The IP address of the Rancher hostname must be resolvable by a public DNS.
+
+   :::
+1. Using the slider, select the number of **Replicas**.
+1. Choose and confirm a **Bootstrap Password**. During the first login, you will use the bootstrap password to authenticate to the Rancher dashboard.
+   :::note
+
+   The current Rancher deployment exposes the bootstrap password in the Cluster configuration settings in Azure Portal. Until this is resolved, we suggest changing the Admin password after initial login by editing your profile in the Rancher dashboard.
+
+   :::
+   ![Rancher Configuration](/img/install-rancher-prime-bootstrap-password.png)
```

Comment:
Is the image intended to highlight 
- The bootstrap password (in which case the alt text needs to be changed)?
- The general Rancher configuration (in which case the image probably shouldn't be nested under the password step)?